aries-site/src/examples/templates/filtering/FilteringWithSelect.js

```diff
@@ -33,322 +25,46 @@ const allData = [
   { name: 'california_1', status: 'Paused' },
 ];
 
-const defaultSelectValue = 'All';
-const defaultFilters = {};
-
-const StyledTextInput = styled(TextInput).attrs(() => ({
-  'aria-labelledby': 'search-icon',
-}))``;
-
-export const FilteringWithSelect = ({ containerRef }) => {
-  // containerRef above is for demo purposes only, remove in production
-  const [data, setData] = useState(allData);
-  const [filtering, setFiltering] = useState(false);
-  const [filters, setFilters] = useState(defaultFilters);
-  const [searchFocused, setSearchFocused] = useState(false);
-  const [search, setSearch] = useState('');
-  const inputRef = useRef();
+export const FilteringWithSelect = () => {
   const size = useContext(ResponsiveContext);
-
-  useEffect(() => {
-    if (searchFocused && inputRef.current) {
-      inputRef.current.focus();
-    }
-  }, [searchFocused, setSearchFocused]);
-
-  const filterData = (array, criteria, searchValue = search) => {
-    if (Object.keys(criteria).length) setFiltering(true);
-    else setFiltering(false);
-    setFilters(criteria);
-
-    let filterResults;
-    const filterKeys = Object.keys(criteria);
-    filterResults = array.filter(item =>
-      // validates all filter criteria
-      filterKeys.every(key => {
-        // ignores non-function predicates
-        if (typeof criteria[key] !== 'function') return true;
-        return criteria[key](item[key]);
-      }),
-    );
-
-    if (searchValue) {
-      filterResults = filterResults.filter(o =>
-        Object.keys(o).some(
-          k =>
-            typeof o[k] === 'string' &&
-            o[k].toLowerCase().includes(searchValue.toLowerCase()),
-        ),
-      );
-    }
-    return filterResults;
-  };
-
-  useEffect(() => {
-    if (filters !== defaultFilters) {
-      setFiltering(true);
-    } else {
-      setFiltering(false);
-    }
-  }, [filters, setFiltering]);
-
   return (
     <Box
```

Comment:
Can we refactor this example to use Page and PageContent? instead of Box with all of these properties?